src/components/Map/components/CardSearchDetails.tsx

```diff
@@ -19,18 +25,41 @@ const CardSearchDetails = ({ result, onClick, onClickClose }: SearchResult) => {
   const { distance } = result.addressDetails?.networkDetails?.network || {};
   const { isEligible } = result.addressDetails?.networkDetails || {};
 
-  const onClickHandler = (ev: React.MouseEvent<HTMLElement>) => {
-    const returnVal =
-      (typeof onClick === 'function' && onClick(result)) || undefined;
-    ev.stopPropagation();
-    return returnVal;
-  };
+  const [contactFormVisible, setContactFormVisible] = useState(false);
 
-  const onCloseHandler = () => {
-    if (onClickClose) {
-      onClickClose(result);
+  const readableDistance = useMemo(() => {
+    if (distance < 1) {
+      return '< 1m';
     }
-  };
+    if (distance >= 1000) {
+      return `${distance / 1000}km`;
```

Comment:
You should probably use a rounding

Reply:
On reflection: 
We get the distance in Meter unit (in Int value) and, at max, we will have equivalent to Number.toFixed(3) and it is the minimum interesting round value. (10 meters can change the result of eligibility test)